app/models/katello/purpose_status.rb

```diff
@@ -38,6 +38,7 @@ def self.to_label(status)
     end
 
     def self.to_status(status, purpose_method, options)
```

Comment:
In this class there's a 'relevant?' method which needs to update its logic to check whether the host's org is in SCA mode. In that case it should return false

Reply:
Some extra credit reading would be to see how `relevant?` is used in Foreman